### PR TITLE
Update tls and ingress samples to properly include ingress class

### PR DESCRIPTION
**Pull request by BryanEllington**

### CODE REVIEW

File: samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml [hunk 1]
```diff
@@ -48,6 +50,8 @@ kibana:
 #         nginx.ingress.kubernetes.io/rewrite-target: /elasticsearch
 #         nginx.ingress.kubernetes.io/backend-protocol: HTTPS
 #       enabled: true
+#       # ingressClass is not currently supported by the Open Distro helm chart
+#       # ingressClassName: nginx
 #       path: /elasticsearch
```
Comment (gsmith-sas) [inline samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml:55]: The comments on lines 53 and 54 are redundant, they duplicate the existing comments on lines 43 and 44.

File: samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml [hunk 2]
```diff
@@ -28,6 +28,8 @@ kibana:
         rewrite (?i)/kibana$ / break;
       nginx.ingress.kubernetes.io/rewrite-target: /kibana
     enabled: true
+    # ingressClass is not currently supported by the Open Distro helm chart
+    # ingressClassName: nginx
```
Comment (gsmith-sas) [inline samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml:32]: Comments on lines 31 and 32 are redundant, duplicating the existing comments on lines 21 and 22.

**Review by gsmith-sas — CHANGES_REQUESTED** [review]
Looks fine other than the redundant comments in the path-based ingress logging sample (see my comments below).

File: samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml (hunk 1 above)
Comment (BryanEllington) [reply samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml:51]: fixed

File: samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml (hunk 2 above)
Comment (BryanEllington) [reply samples/ingress/path-based-ingress/logging/user-values-elasticsearch-open.yaml]: fixed

**Review by gsmith-sas — APPROVED** [review]
LGTM

**Pull request merged**